### PR TITLE
Add deephaven.learn/NumPy notebook for live demo

### PR DESCRIPTION
**Pull request by jjbrosnan**
This notebook uses deephaven.learn and NumPy to show a prospective user how they work in Deephaven.

### CODE REVIEW

File: demo/web/src/main/notebooks/Python_AI/00 Deephaven with PyTorch.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+# Deephaven with PyTorch
```
Comment (margaretkennedy) [inline demo/web/src/main/notebooks/Python_AI/00 Deephaven with PyTorch.md:1]: B. Data science, ML, & AI "B4 PyTorch and Deephaven"

File: demo/web/src/main/notebooks/Python_AI/01 Deephaven with SciKit-Learn.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
+# Deephaven with SciKit-Learn
```
Comment (margaretkennedy) [inline demo/web/src/main/notebooks/Python_AI/01 Deephaven with SciKit-Learn.md:1]: B. Data science, ML, & AI / B5 "B5 ScikitLearn and Deephaven"

File: demo/web/src/main/notebooks/Python_AI/02 Deephaven with TensorFlow.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
+# Deephaven with TensorFlow
```
Comment (margaretkennedy) [inline demo/web/src/main/notebooks/Python_AI/02 Deephaven with TensorFlow.md:1]: B. Data science, ML, & AI "B3 TensorFlow and Deephaven"

**Review by margaretkennedy — CHANGES_REQUESTED** [review]
Just file names

**Comment by jjbrosnan** [comment]
This PR is being closed for a newer one with less differences in file structure:

[Numba + NumPy](https://github.com/deephaven/deephaven-core/pull/1934).

**Pull request closed without merging**